src/edu/csus/ecs/pc2/core/InternalController.java

```diff
@@ -1002,6 +1002,8 @@ public void initializeServer(IInternalContest inContest) throws IOException, Cla
             
             handleCDPLoad(true);
             
+            loadDefaultJudgementsIfMissing();
```

Comment:
The total description of the Issue (#119) to which this PR applies says:
`There are non-logged exceptions.
There are compiler warnings in the project.
`
It does not seem to me that addition of a whole new method, which has potentially significant logic ramifications, falls under Issue #119.  This should have been submitted as a separate issue/PR.

Reply:
I did not make this change.

Reply:
seems your fork may have been out of date in that case.